plugins/PrivacyManager/lang/en.json

```diff
@@ -161,13 +161,17 @@
         "UseAnonymizeTrackingData": "Anonymize Tracking Data",
         "UseAnonymizeUserId": "Anonymize User ID",
         "PseudonymizeUserId": "Replace User ID with a pseudonym",
-        "OptOutCustomize": "Customize the Opt-out iframe",
+        "OptOutCustomize": "Customize the opt-out appearance",
```

Comment:
You should use a new translation key here. The meaning of the content changed too much. Existing translations for other languages would otherwise still be used and might cause confusion.

Reply:
Good spot, thanks :slightly_smiling_face: